citations/citations-api/api/src/java/org/sakaiproject/citation/api/CitationService.java

```diff
@@ -248,6 +248,25 @@
 	 * @param citationCollectionId
 	 */
 	public CitationCollection getUnnestedCitationCollection(String citationCollectionId);
+
+	/**
+	 ** Gets a citation collection including nested and unnested citations
+	 * @param citationCollectionId
+	 */
+	public List<CitationCollectionOrder> getNestedCollectionAsList(String citationCollectionId);
```

Comment:
Why do these need to be in the API as they are just used inside the citation service implementation?